src/ResourceManager/ManagedServiceIdentity/Commands.ManagedServiceIdentity/help/Get-AzureRmUserAssignedIdentity.md

```diff
@@ -0,0 +1,229 @@
+---
```

Comment:
Be sure to regenerate the help after fixing the parameter sets.

Reply:
done

Reply:
You'll need to do this one more time (to pick up the rename from Identity to InputObject)

Reply:
done